src/cpu/core_dyn_x86/risc_x64.h

```diff
@@ -303,7 +304,7 @@ static BlockReturn gen_runcodeInit(Bit8u *code) {
 	opcode(0).setea(4,-1,0,CALLSTACK).Emit8(0x89);  // mov [rsp+8/40], eax
 	opcode(4).setrm(ARG0_REG).Emit8(0xFF);   // jmp ARG0
 
-	*(Bit32u*)diff = (Bit32u)(cache.pos - diff - 4);
```

Comment:
PPC: Fix needs to be aligned and native endian (not swapped to little-endian like the host_* functions do).